src/api/localStorage.js

```diff
@@ -0,0 +1,93 @@
+export class LocalStorage {
+  constructor() {
+    this.users = JSON.parse(localStorage.getItem('userAccounts'));
```

Comment:
Good use of the `constructor`, I would go a bit further though. You can't be certain you'll have anything to parse, so you'll want to account for that.
```suggestion
    const storageUsers = localStorage.getItem('userAccounts');
    this.users = storageUsers ? JSON.parse(storageUsers) : [];
```
if you're not familiar with the **_prop_ ? _statement_ : _statement_** syntax, give [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) a look over. Conditional operator/ternary expressions are super handy. That said, you could also (probably) simplify it further by just using an `||` statement
```suggestion
    this.users = JSON.parse(localStorage.getItem('userAccounts')) || [];
```
This _should_ evaluate the first expression (`JSON.parse(localStorage.getItem('userAccounts'))`), and if false, sets `this.users` to the expression on the other side of the `||`. Also pretty nifty.